receiver/jmxreceiver/internal/subprocess/subprocess_mock.go

```diff
@@ -0,0 +1,57 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Since this is test code, let's name it with the `_test.go` suffix, so it doesn't count against code coverage.

Reply:
changed